app/controllers/Controller.scala

```diff
@@ -100,16 +111,31 @@ abstract class ApplicationController()
     }
   }
 
-  def subFieldsDefinitionsExistAction(applicationId: String)
+  private object ManageSubscriptionsActions {
+    def subscriptionsStackedActions(applicationId: String, noFieldsBehaviour : NoSubscriptionFieldsRefinerBehaviour)
```

Comment:
Stacked is a t2v concept - maybe Composed might be better ?

Reply:
Just to clarify (as we're not using t2v), is this just a rename 'stacked' to 'compose'? If so, yup makes sense.